pkg/core/helper_test.go

```diff
@@ -375,3 +391,17 @@ func TestCreateBasicChain(t *testing.T) {
 		}
 	}
 }
+
+func newNEP5Transfer(sc, from, to util.Uint160, amount int64) *transaction.Transaction {
+	w := io.NewBufBinWriter()
+	emit.Int(w.BinWriter, amount)
+	emit.Bytes(w.BinWriter, to.BytesBE())
+	emit.Bytes(w.BinWriter, from.BytesBE())
+	emit.Int(w.BinWriter, 3)
+	emit.Opcode(w.BinWriter, opcode.PACK)
+	emit.String(w.BinWriter, "transfer")
+	emit.AppCall(w.BinWriter, sc, true)
```

Comment:
Shouldn't there be a `THROWIFNOT` after this?

Reply:
Why do we need it?
THROW will happen inside a contract if needed.

Reply:
I've seen it many times in real invocations, it makes transaction fail which I think is fair, as per [NEP-5 spec:](https://github.com/neo-project/proposals/blob/master/nep-5.mediawiki)
```
If the method succeeds, it MUST fire the transfer event, and MUST return true, even if the amount is 0, or from and to are the same address. 
...
If the transfer is not processed, the function SHOULD return false. 
```
And actually, you're not supposed to THROW anything from the contract in case of insufficient funds.